### ISSUE TITLE
Calling configure multiple times overwrites previous register variable

### ISSUE THREAD

**Issue by woodcockjosh** [mashpie/i18n-node#437]
I have 2 json files in different directories
`locales/a/eng-US.json`
```json
{
	"key": "A"
}
```

`locales/b/eng-US.json`
```json
{
	"key": "B"
}
```

`test.js`
```javascript
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
const i18n = require("i18n");
const i18nDataUnsafeA = {};
i18n.configure({
    locales: ['eng-US'],
    directory: `${__dirname}/locales/a`,
    register: i18nDataUnsafeA
});
const i18nDataA = Object.assign({}, i18nDataUnsafeA);
const i18nDataUnsafeB = {};
i18n.configure({
    locales: ['eng-US'],
    directory: `${__dirname}/locales/b`,
    register: i18nDataUnsafeB
});
const i18nDataB = Object.assign({}, i18nDataUnsafeB);
console.log(i18nDataA.__({ phrase: 'key', locale: 'eng-US' }));
console.log(i18nDataB.__({ phrase: 'key', locale: 'eng-US' }));
```

Output
```
B
B
```
Why is that???? How can I prevent my language objects from being overridden? I have a library which uses its own language files and a web app that uses that library. The web application is overriding the library's language file objects.

**Comment by mashpie** [comment]
FTR: as you already spotted... i18n is a singleton - one work around would be to use https://www.npmjs.com/package/import-fresh instead of require()

**Comment by mashpie** [comment]
Instance feature is released as of 0.12.0 with some extra sugar (shortcut to i18n.configure on creation):

```js
const { I18n } = require("i18n");

const i18n = new I18n({
    locales:['en', 'de'],
    directory: __dirname + '/locales'
});
```

see https://github.com/mashpie/i18n-node/tree/master#as-instance for details
